app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizReviewActivity.java

```diff
@@ -32,34 +37,46 @@ protected void onCreate(Bundle savedInstanceState) {
         try {
             all_forgotten_cards = this.getIntent().getParcelableArrayListExtra(FORGOT_CARDS);
             all_remembered_cards = this.getIntent().getParcelableArrayListExtra(REMEMBERED_CARDS);
+            quizScore = this.getIntent().getStringExtra(QUIZ_SCORE);
         }
         catch (NullPointerException e) {
```

Comment:
instead of catching nullpointer we should probably just check if forgotten and rememebered cards are null

Reply:
removed